go/hyperkit.go

```diff
@@ -367,32 +375,49 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	a = append(a, "-m", fmt.Sprintf("%dM", h.Memory))
 
 	a = append(a, "-s", "0:0,hostbridge")
+	a = append(a, "-s", "31,lpc")
+
+	nextSlot := 1
```

Comment:
It's to be able to add multiple entries for some parameters, such as disk, or 9p mounts in this case. Without it we would only support one or zero (or a fixed number of dedicated slots).